.github/workflows/rust.yml

```diff
@@ -65,6 +65,7 @@ jobs:
         LINKER: "lld-link.exe"
         CC: "clang-cl.exe"
         CXX: "clang-cl.exe"
+        NATIVE_WIN32_PYTHON: "C:\\mozilla-build\\python2\\python.exe"
```

Comment:
Why is this required?

Reply:
https://github.com/servo/mozjs/blob/master/makefile.cargo#L169